const foo = 1;

//this test should  still fail
